Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:14.15.3
+FROM node:latest
```

Comment:
Like mentioned earlier I think we should pin a version just for consistency

Reply:
Updated to `node:lts`.